spec/integ/matrix-client-syncing.spec.ts

```diff
@@ -1990,5 +1990,8 @@ describe("MatrixClient syncing (IndexedDB version)", () => {
         idbHttpBackend.verifyNoOutstandingExpectation();
         await idbClient.stopClient();
         await idbHttpBackend.stop();
+
+        // force a save to indexeddb to make sure that we don't have any saves in flight when the test completes.
+        await idbClient.store.save(true);
```

Comment:
once we limit jest to one worker, it realises that something is logging after shutdown, and complains about it. This makes sure that the test cleans up properly.